wfl/Models/MenuSelection.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.IdGenerators;
+
+namespace Symphono.Wfl.Models
+{
+    public class MenuSelection: IEntity
+    {
+        [BsonIgnoreIfNull]
+        [BsonId(IdGenerator = typeof(StringObjectIdGenerator))]
+        public string Id { get; set; }
+        public string OrdererName { get; set; }
+        public string Description { get; set; }
+        public string FoodOrderId { get; set; }
```

Comment:
That makes a ton of sense...this change also goes hand in hand w/ the URL change I asked about [here](https://github.com/Symphono/wfl/pull/41#discussion_r128103837)...